crates/pacdef_core/src/backend/actual/python.rs

```diff
@@ -57,9 +67,18 @@ fn run_pip_command(args: &[&str]) -> Result<Value> {
 impl Python {
     pub(crate) fn new() -> Self {
         Self {
+            binary: BINARY.to_string(),
             packages: HashSet::new(),
         }
     }
+
+    fn get_switches_runtime(&self) -> Switches {
+        if self.get_binary().eq("pip") {
```

Comment:
please `match` the String against `pip` and `pipx`, and `panic!()` if the user supplies anything else.

Reply:
> please `match` the String against `pip` and `pipx`, and `panic!()` if the user supplies anything else.

This has been implemented in the recent commit. Please have a look.